modules/video/stream/stream.go

```diff
@@ -3,26 +3,90 @@ package stream
 import (
 	"context"
 	"errors"
+	"time"
 
+	"github.com/NTHU-LSALAB/NTHU-Distributed-System/modules/video/dao"
 	"github.com/NTHU-LSALAB/NTHU-Distributed-System/modules/video/pb"
+	"github.com/NTHU-LSALAB/NTHU-Distributed-System/pkg/kafkakit"
+	"github.com/golang/protobuf/proto"
 	"github.com/justin0u0/protoc-gen-grpc-sarama/pkg/saramakit"
+	"go.mongodb.org/mongo-driver/bson/primitive"
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
 type stream struct {
 	pb.UnimplementedVideoStreamServer
+
+	producer kafkakit.Producer
+	videoDAO dao.VideoDAO
```

Comment:
```suggestion
	videoDAO dao.VideoDAO
	producer kafkakit.Producer
```

跟 `NewStream` 時 declare 的順序一樣比較好